formula-test/src/test/java/com/instacart/formula/test/TestActionObserverTest.kt

```diff
@@ -0,0 +1,61 @@
+package com.instacart.formula.test
+
+import com.google.common.truth.Truth
+import com.google.common.truth.Truth.assertThat
+import com.instacart.formula.Action
+import com.instacart.formula.Cancelable
+import org.junit.Test
+import java.lang.IllegalStateException
+
+class TestActionObserverTest {
+
+    @Test fun `assert values success`() {
+        multipleValueStream().test().assertValues(1, 2)
+    }
+
+    @Test fun `assert value fails due to different size`() {
+        val result = runCatching { multipleValueStream().test().assertValues(1) }
+        assertThat(result.exceptionOrNull()).isInstanceOf(AssertionError::class.java)
+    }
+
+    @Test fun `assert value fails due to different value`() {
+        val result = runCatching { multipleValueStream().test().assertValues(1, 5) }
+        assertThat(result.exceptionOrNull()).isInstanceOf(AssertionError::class.java)
+    }
+
+    @Test fun values() {
+        val values = multipleValueStream().test().values()
+        assertThat(values).hasSize(2)
+    }
+
+    @Test fun `cancel throws exception if action does not provide cancelable`() {
+        val result = kotlin.runCatching { multipleValueStream().test().cancel() }
```

Comment:
Is there a difference between above `runCatching` block vs `kotlin.runCatching`?

Reply:
It's the same function, the auto-complete keeps giving me `kotlin.runCatching` instead of `runCatching`

Reply:
Oh! That's so weird